terraform/dev/main.tf

```diff
@@ -26,8 +30,7 @@ provider "cloudfoundry" {
 }
 
 provider "aws" {
-  region  = var.aws_region
-  version = "~> 3.11.0"
```

Comment:
I got some deprecation warnings that version constraints had to be in the `required_providers` section, so that's why these got moved.